src/main/java/org/example/db/DeliveryEmployeeDao.java

```diff
@@ -60,4 +61,26 @@ public int createDeliveryEmployee(DeliveryEmployeeRequest employee) throws SQLEx
 
         return -1;
     }
+
+    public DeliveryEmployee getDeliveryEmployeeById(int id) throws SQLException {
+        Connection c = databaseConnector.getConnection();
+        String selectStatement  = "SELECT delivery_employee_id, `name`, salary, bank_account_number, national_insurance_number FROM DeliveryEmployee WHERE delivery_employee_id = ?;";
+
+        PreparedStatement st = c.prepareStatement(selectStatement, Statement.RETURN_GENERATED_KEYS);
```

Comment:
Is `Statement.RETURN_GENERATED_KEYS` needed?

Reply:
Removed.